util/module_logger_test.go

```diff
@@ -0,0 +1,253 @@
+package util_test
+
+import (
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"github.com/uber-common/bark"
+	"github.com/uber/ringpop-go/test/mocks"
+	"github.com/uber/ringpop-go/util"
+	"testing"
+)
+
+type MockLogger struct {
```

Comment:
What does a `MockLogger` do that's different to `mocks.Logger`? Can we document this a comment? Also, does it need to be exported (I am thinking not)?